modules/claims_api/config/schemas/526.json

```diff
@@ -309,10 +309,8 @@
         "homelessness": {
           "description": "Object describing Veteran Homelessness if applicable.",
           "type": "object",
+          "required": ["pointOfContact"],
```

Comment:
Yes, this is in the [EVSS Documentatio](https://www.ebenefits.va.gov/wss-form526-services-web-v2/swagger-ui/index.html?url=https://www.ebenefits.va.gov/wss-form526-services-web-v2/rest/swagger.yaml#/form526/validate)n and something previously missing on our end. I verified with the staging validate endpoint and observed the following error when including the `homeless` attribute:

```json
        {
            "status": 422,
            "detail": "form526.veteran.homelessness.pointOfContact.NotNull ",
            "source": "form526.veteran.homelessness.pointOfContact.NotNull"
        },
```

Reply:
Sorry to keep this one going, but I was going to mention that we talked about this in stand-up this morning while you were out and we were concerned about changing the contract here. 

Can we remove this change in this PR and only have this PR worry about passing the single character when necessary?  

If we find that we need to come back and break the contract for `pointOfContact` then we can handle that in a future ticket where we can also make sure to give our consumers plenty of notice that the contract is going to change.

Reply:
To be clear, it's only required if they supply the top-level property of `homelessness`, so it won't show that error is other cases.

I think we should include this validation because omitting it will cause a failure at the EVSS level anyway, so this is functioning much like a lot of the existing 526 validations. Also, unlike some of the other values where it's easy to silently transform to an empty string or something trivial, there's not really a good way to supply some filler data to get a submission through without these values.

Reply:
>it's only required if they supply the top-level property of homelessness


Right, but the potential prob is that there could be someone out there who is currently sending us `homelessness` but they aren't providing `pointOfContact` for some reason.  (IDK why, but possible).  If that scenario is happening, this would begin to break their implementation.

Reply:
Unless I'm misreading things, `pointOfContact` is conditionally required already.  
So if they provide us either `currentlyHomeless` or `homelessnessRisk`, then they have to give us `pointOfContact`.
So do we really need to make `pointOfContact` a hard-requirement?

Reply:
per EVSS discussions, I've removed the `pointOfContact` requirement in the schema until things are resolved and/or answered on their end ([ref](https://dsva.slack.com/archives/C8R3JS8BU/p1655136820013309?thread_ts=1654890270.943629&cid=C8R3JS8BU))

Reply:
Should we not leave our contract as-is with the `dependentRequired` for `pointOfContact`?
It sounds like EVSS still requires it as of now, so I guess I'm confused on why we're making it completely optional on our end now.

Reply:
I added those validations within the controller [here](https://github.com/department-of-veterans-affairs/vets-api/blob/52e16c91c67bb09c577ba92096884b910975c50a/modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb#L200-L214), because the error messaging generated via `dependentRequired` wasn't helpful at all. It basically just said "this #{request body} value for `homelessness` is invalid", with no clear remedy for fixing it